backend/src/main/java/com/woowacourse/kkogkkog/presentation/MemberController.java

```diff
@@ -52,6 +52,13 @@ public ResponseEntity<MemberHistoriesResponse> showHistory(@LoginMember Long id)
         return ResponseEntity.ok(new MemberHistoriesResponse(memberService.findHistoryById(id)));
     }
 
+    @PutMapping("/me/histories")
+    public ResponseEntity<Void> updateAllMemberHistories(@LoginMember Long id) {
```

Comment:
`Put`은 아닌 것 같아요. API 명세를 수정하는 것은 어떨까요? URI를 수정해도 괜찮고요 

```suggestion
    @PatchMapping("/me/histories")
    public ResponseEntity<Void> updateAllMemberHistories(@LoginMember Long id) {
```